Pages/AlertPage.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Atata;
+
+namespace IFlow.Testing.Pages
+{
+    using _ = AlertPage;
+    [Url("https://demoqa.com/alerts")]
+    [VerifyTitle("ToolsQA")]
+    public class AlertPage : BasePage<_>
+    {
+        [FindById("alertButton")]
+        public Button<_> AlertButton { get; set; }
+
+        [FindById("confirmResult")]
+        public Text<_> ResultText { get; set; }
+
+        [FindById("confirmButton")]
+        public Button<_> Cancel { get; set; }
+
+        public void DissmisAlert()
```

Comment:
Z tego wynika że to tak na prawdę są klasy pomocnize. Nie odwołują sie do obiektów w danym miejscu ale do obiektów mogących wystąpić wszędzie w aplikacji. Są też dość uniwersalne. Proponuje stworzyć odpowiednia klase w stylu helpers w folderze utils albo tam znaleźć miejsce